apps/aecore/lib/aecore/naming/tx/name_transfer_tx.ex

```diff
@@ -44,8 +45,12 @@ defmodule Aecore.Naming.Tx.NameTransferTx do
   # Callbacks
 
   @spec init(payload()) :: t()
+  # TODO integrate Identifiers
```

Comment:
is this todo still to be done?

Reply:
This is just a part of unused comments, will remove them soon.